modules/nf-core/fastqc/tests/main.nf.test

```diff
@@ -21,12 +25,12 @@ nextflow_process {
         }
 
         then {
-            assert process.success
-            assert process.out.html.get(0).get(1) ==~ ".*/test_fastqc.html"
-            assert path(process.out.html.get(0).get(1)).getText().contains("<tr><td>File type</td><td>Conventional base calls</td></tr>")
-            assert process.out.zip.get(0).get(1) ==~ ".*/test_fastqc.zip"
+            assertAll (
+            { assert process.success },
+            { assert snapshot(process.out.html).match("html") },
```

Comment:
I don't think you can md5sum the html report

Reply:
That's so weird. Why did we never test it with pytest-workflow?

Oh well, good enough for me!